src/convolver.cc

```diff
@@ -114,7 +115,7 @@ TimeDomainConvolver::configure (Readable* r, float gain, uint32_t delay)
 	r->read (&_ir[delay], 0, to_read, 0);
 
 	if (gain != 1.f) {
-		for (uint64_t i = delay; i < 64; ++i) {
+		for (uint32_t i = delay; i < _ir_len; ++i) {
```

Comment:
`_ir_len` is undefined, maybe something is missing from the PR here?
The constant 64  here is intentional, without it, various places may need a ` pragma GCC ivdep`

Reply:
`ir_len` is fined by the patch a bit above, per:

```
const uint32_t _ir_len = sizeof (_ir) / sizeof (float);
```

Reply:
we can also use constexpr if having a compiler-time value is needed.

Reply:
C++98 please (I'm still doing OSX builds), so constexpr is out. 
Also since it is not a private member variable there should not be a leading underscore. This is why I missed it

Reply:
I was not aware you had a convention of using underscore of 1st char as indication of private members, good to know!

I can do a little extra patch to turn the new var into a macro, which allows us to keep old optimization

Reply:
I considered it, but it's really just in `configure`. so it's fine.